ruby/vampires.rb

```diff
@@ -10,14 +10,15 @@
 
 until number_of_hires <= 0
 	puts "---"
-	# Gets User name
+	# Gets hire's name
 	puts "What is your name?"
 	name = gets.chomp
 
-	# Gets user age
+	# Gets hire's age
 	puts "How old are you?"
 	age = gets.chomp.to_i
 
+	#Gets hire's date of birth
 	puts "What year were you born?"
 	birthyear = gets.chomp.to_i
 	realage = 2016 - birthyear

```

Comment:
Look at the time class in the Ruby docs to fine `Time.now.year` and see if you can find a dynamic way to add this to your code to make sure that we don't have to rewrite this file every year.